pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,388 @@
+
+Bento components are available experimentally and you must [enable experimental features](../learn/experimental.md).
+
+You can enable all Bento components by including the script below:
+
+
+```
+<script>
+  (self.AMP = self.AMP || []).push(function (AMP) {
+    AMP.toggleExperiment('bento', true);
+  });
+</script>
+```
+
+
+Or enable a single Bento component by including the script below. Ensure you replace `bento-foo` with the desired [supported component](#available-bento-components) name given on its documentation page:
+
+
+```
+<script>
+  (self.AMP = self.AMP || []).push(function (AMP) {
+    AMP.toggleExperiment('bento-foo', true);
+  });
+</script>
+```
+
+
+
+## Import AMP runtime and Bento component logic
+
+You must include the AMP runtime script and import the logic for each individual Bento component desired.
+
+```
+<script async src="https://cdn.ampproject.org/v0.js"></script>
+
+<script async custom-element="amp-bento-component-name" src="https://cdn.ampproject.org/v0/amp-bento-1.0.js"></script>
+``` 
+
+Read each Bento component’s reference documentation for implementation details.
+
+
+# Available Bento components  <a name="available-bento-components"></a>
+
+Bento supported AMP components are listed below:
+
+
+
+*   [amp-accordion](../../../documentation/components/reference/amp-accordion.md)
+*   [amp-base-carousel](../../../documentation/components/reference/amp-base-carousel.md)
+*   [amp-inline-gallery](../../../documentation/components/reference/amp-inline-gallery.md)
+    *   amp-inline-gallery-pagination
+    *   amp-inline-gallery-thumbnails
+*   [amp-stream-gallery](../../../documentation/components/reference/amp-stream-gallery.md)
+*   [amp-date-countdown](../../../documentation/components/reference/amp-date-countdown.md)
+*   [amp-date-display](../../../documentation/components/reference/amp-date-display.md)
+*   [amp-fit-text](../../../documentation/components/reference/amp-fit-text.md)
+*   [amp-instagram](../../../documentation/components/reference/amp-instagram.md)
+*   [amp-lightbox](../../../documentation/components/reference/amp-lightbox.md)
+*   [amp-selector](../../../documentation/components/reference/amp-selector.md)
+*   [amp-social-share](../../../documentation/components/reference/amp-social-share.md)
+*   [amp-timeago](../../../documentation/components/reference/amp-timeago.md)
+*   Video components
+    *   [amp-video](../../../documentation/components/reference/amp-video.md)
+    *   [amp-youtube](../../../documentation/components/reference/amp-youtube.md)
+
+# Bento in action
+
+The example below demonstrates how to include `amp-base-carousel` and `amp-fit-text` in a non-AMP HTML page.
+
+```
+<!DOCTYPE html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8" />
+    <title>AMP Public Radio</title>
+    <link rel="canonical" href="self.html" />
+    <meta
+      name="viewport"
+      content="width=device-width,minimum-scale=1,initial-scale=1"
+    />
+    <style amp-boilerplate>
```

Comment:
Mind shrinking the boilerplate here? Looks like prettier might have expanded it.

Reply:
I was thinking of removing it - is that still valid for the experiment, to just have the runtime script?

Reply:
Yes, that works!

Reply:
Is the boilerplate needed with bento?